src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -57,8 +57,29 @@
 using namespace cldnn;
 
 void prepare_primitive_fusing::run(program& p) {
-    GPU_DEBUG_IF(p.get_config().get_disable_post_ops_fusions())
-        return;
+    GPU_DEBUG_IF(p.get_config().get_disable_post_ops_fusions() != 0) {
```

Comment:
Honestly, I'm not sure if these many steps are necessary. For most fusion-related issues for me, I think it's more necessary to separate quantize, eltwise, and activation inside in fuse_simple_primitives.

Reply:
I think this part of code needs some additional clean-up. Could you separate the PR for original code / debugging code?

Reply:
Removed debugging feature.